pkg/kubelet/dockertools/labels.go

```diff
@@ -67,6 +70,11 @@ func newLabels(container *api.Container, pod *api.Pod, restartCount int) map[str
 	labels[kubernetesContainerRestartCountLabel] = strconv.Itoa(restartCount)
 	labels[kubernetesContainerTerminationMessagePathLabel] = container.TerminationMessagePath
 
+	// Add annotation for cAdvisor to handle custom metrics if required.
+	if enableCustomMetrics {
+		addCustomMetricsLabelIfRequired(pod, labels)
```

Comment:
Why introducing a new docker label here? Why not passing such config knob directly to cAdvisor since cAdvisor is linked with Kubelet anyway? In your PR, the location and file name are both constants, no need to pass as label. 

I don't like to use docker label here because you are relying on a runtime internal implementation, not API, at least it is not align with rkt.